topics/ingest-feedbacks.yaml

```diff
@@ -0,0 +1,17 @@
+topic: ingest-feedbacks
+pipeline: replays
+description: User feedback widget events
+services:
+  producers:
+    - getsentry/sentry
+  consumers:
+    - getsentry/sentry
+schemas:
+  - version: 1
+    compatibility_mode: none
+    type: json
+    resource: events.v1.schema.json
+    examples:
+      - events/1/
```

Comment:
you need to provide a schema as well as examples for your new topic

Reply:
Based on Colton's comment https://github.com/getsentry/sentry/pull/66169#discussion_r1511527232
it sounds a lot like this is supposed to match the schema of `ingest-events`, not `events`

Reply:
If the schema/examples are different or going to diverge, I would suggest making a different file now, but i leave that up to you.

But what i'm specifically saying is that `events` is not the same as `ingest-events` at all. It sounds like today the schema more closely matches `ingest-events` since they are literally combined today. `events` is an entirely different thing.